org-node/src/client.rs

```diff
@@ -359,12 +359,22 @@ impl Client {
                                 continue;
                             }
 
-                            if let Err(err) = Client::sign_refs(urn, &api).await {
+                            if let Err(err) = Client::sign_refs(urn.clone(), &api).await {
                                 tracing::error!(target: "org-node", "Error signing refs: {}", err);
                                 ack.send(Err(err)).ok();
                                 continue;
                             }
 
+                            // Announce the peer update to the network.
+                            if let Err(e) = api.announce(Payload {
+                                urn,
+                                // QUESTION: Should the current revision be added here?
```

Comment:
Yeah, I would say so. Don't quite understand what happens if you don't include a rev, ie. how peers are supposed to interpret that..

Reply:
removed announce statement